com.github.raibtoffoletto.litteris.yml

```diff
@@ -0,0 +1,38 @@
+app-id: com.github.raibtoffoletto.litteris
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+base: io.elementary.BaseApp
+base-version: juno-19.08
+command: com.github.raibtoffoletto.litteris
+finish-args:
+  #X11
+  - --share=ipc
+  - --socket=x11
+  # Wayland
+  - --socket=wayland
+  #Writing to the FS
+  - --filesystem=home:create
```

Comment:
What does it need home access for? Have you tried to use the GtkNativeFileChooser ?

Reply:
It needs to be able to export/import back-ups and to select a cloud-synced folder (Dropbox/Nextcloud,etc) for synchronize between devices.

Reply:
In my code it uses the Gtk.FileChooserNative ;)

Reply:
It should be possible to use GtkFileChooserNative to select folders in the near future; make sure to check that in few months and remove the home permission =) otherwise lgtm

Reply:
The portal works fine for export/import, but for the sync capabilities the Gtk.FileChooserButton and the Environment.get_home_dir to create the path wouldn't work ATM. I'll stay tuned.